app/services/consumption/testOnly/ResourceConfig.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.consumption.testOnly
+
+import play.api.Configuration
+
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+private[testOnly] class ResourceConfig @Inject() (config: Configuration) {
```

Comment:
Again could probably refactor this to have a common get method that takes in the name of the resource. Then we wouldn't need a method for each code list. Not essential for this PR.